man/systemd.netdev.xml

```diff
@@ -2258,6 +2261,56 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[HSR] Section Options</title>
+    <para>The [HSR] section only applies for netdevs of kind <literal>hsr</literal> and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>SlaveInterface1=</varname></term>
```

Comment:
Please do not use "slave" for new settings. See #26257.
How about "Port1" or so?